{"name": "test_getbyname_failure", "status": "passed", "start": 1729284976058, "stop": 1729284976058, "uuid": "86a8f370-ef99-4f7f-987d-580a8b8aa968", "historyId": "a93088beb91554d7a6f791d56bf46900", "testCaseId": "a93088beb91554d7a6f791d56bf46900", "fullName": "Exercise.exercise_test.TestExerciseServiceWithoutMock#test_getbyname_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithoutMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "12956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}